sdk/cosmos/azure_data_cosmos/src/options/mod.rs

```diff
@@ -47,7 +47,7 @@ pub struct DeleteDatabaseOptions<'a> {
 #[derive(Clone, Default)]
 pub struct ItemOptions<'a> {
     pub method_options: ClientMethodOptions<'a>,
-
+    pub if_match_etag: Option<Etag>,
```

Comment:
Let's add [documentation comments](https://doc.rust-lang.org/stable/book/ch14-02-publishing-to-crates-io.html?highlight=documentation#making-useful-documentation-comments) to this, I'm trying to keep us up-to-date on that as we add features :).

I'll let you take a first stab at writing it, take a look at the comments for the other fields (`method_options` doesn't have one yet, but we'll address that later) and see what you think. Happy to help if you're not sure what to write! I'd include a link to this content on our docs site as well: https://learn.microsoft.com/azure/cosmos-db/nosql/database-transactions-optimistic-concurrency#optimistic-concurrency-control , since it describes the `if-match` header in detail.